src/Public/Responses.ps1

```diff
@@ -19,14 +23,23 @@ Set-PodeResponseAttachment -Path './image.png'
 
 .EXAMPLE
 Set-PodeResponseAttachment -Path 'c:/content/accounts.xlsx'
+
+.EXAMPLE
+Set-PodeResponseAttachment -Path './data.txt' -ContentType 'application/json'
+
+This command specifies that the content stored in data.txt should be considered as JSON data rather than plain text.
 #>
 function Set-PodeResponseAttachment
 {
     [CmdletBinding()]
     param (
         [Parameter(Mandatory=$true)]
         [string]
-        $Path
+        $Path,
+
+        [ValidatePattern('\w+\/[\w.-]+')]
```

Comment:
Some valid content types have a `+`, such as: `application/windows-search-connector+xml`. There's a massive list of content types within the `Private/Mappers.ps1` file. (when I added the `+` it appeared to match them all - I'd also advise escaping the `.` in case it starts matching every character on some platforms).

Ah, I'd also recommend adding `^` to the start, and `$` to the end - to avoid erroneous text around the type.

Reply:
Good catches, thanks. I misses the + when looking through the list (I actually used that mappers list to generate a list for ValidateSet... yeah, way too long.)

Commit incoming.